### PR TITLE
EPIC: ESL Select & ESL Select List (beta)

### PR DESCRIPTION
**Pull request by ala-n**
- Support for single selection
- Both select and select list moved under forms directory
- Beta doc placeholder created

### CODE REVIEW

File: src/modules/esl-forms/esl-select-list/core/esl-select-list.less [hunk 1]
```diff
@@ -22,12 +22,16 @@
...
 
-  esl-scrollbar.esl-scrollbar {
-    top: 60px;
+  esl-scrollbar {
+    top: 5px;
     bottom: 5px;
     right: 10px;
   }
 
+  &[multiple] esl-scrollbar{
```
Comment (yadamskaya) [inline src/modules/esl-forms/esl-select-list/core/esl-select-list.less:31]: You lost " " here

File: src/modules/esl-forms/esl-select/core/esl-select-dropdown.less [hunk 2]
```diff
@@ -30,10 +30,14 @@
...
 
-    esl-scrollbar.esl-scrollbar {
-      top: 56px;
+    esl-scrollbar {
+      top: 6px;
       bottom: 6px;
       right: 2px;
     }
+
+    &[multiple] esl-scrollbar{
```
Comment (yadamskaya) [inline src/modules/esl-forms/esl-select/core/esl-select-dropdown.less:39]: and here

**Review by julia-murashko — APPROVED** [review]

**Review by yadamskaya — APPROVED** [review]

**Review by andreybelous — APPROVED** [review]

**Pull request merged**